mm2src/coins/lp_coins.rs

```diff
@@ -3501,7 +3501,7 @@ impl DexFee {
 pub struct CoinsContext {
     /// A map from a currency ticker symbol to the corresponding coin.
     /// Similar to `LP_coins`.
-    coins: AsyncMutex<HashMap<String, MmCoinStruct>>,
+    pub coins: AsyncMutex<HashMap<String, MmCoinStruct>>,
```

Comment:
Please add a function instead of giving a public access to this field.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2100/commits/350abd681208c9bccfec5a03a8b93ba19af694d6
created pub function to lock coins